.github/pull-request-template.md

```diff
@@ -1,4 +1,22 @@
-Fixes #(issue-number)
+<!-- PR tips
+
+New contributors:
+
+If you are new to Git/GitHub and want to make a quick fix to the docs,
+open your PR against the release branch where you found the error, such as
+"release-0.5".
+
+Experienced contributors:
```

Comment:
```suggestion
Experienced/regular contributors:
```
Maybe something along these lines to qualify that you dont need to have a "level of experience" but rather this is for someone who regularly contributes to knative?

Reply:
Changed to just "regular"